CONTRIBUTING.md

```diff
@@ -29,13 +35,30 @@ for development
     ```sh
     git clone https://github.com/opencv/cvat
     cd cvat && mkdir logs keys
-    python3 -m venv .env
+    python -m venv .env
     . .env/bin/activate
     pip install -U pip wheel setuptools
     pip install -r cvat/requirements/development.txt
     python manage.py migrate
     python manage.py collectstatic
     ```
+    > Note for Mac users
+    >
+    > You may need to reinstal python as a first quick solution if you are faced with errors on step
```

Comment:
I'm not comfortable to advise users delete files. Could you please provide a symptom and a possible solution (general idea of the solution)?

For example, if you have any problems with installing dependencies from cvat/requirements/*.txt files, you may need to reinstall your system python. In some cases after system update it can be configured incorrectly and cannot compile some native modules.